com.github.ransome1.sleek.yml

```diff
@@ -3,7 +3,7 @@ runtime: org.freedesktop.Platform
 runtime-version: '23.08'
```

Comment:
Oh I think you might need to update this as well. BaseApp version and runtime version seem to be coupled together.

Reply:
@Lezurex Since we are not building the Electron app during the build process, but instead bundling ready-made Debian files, I wonder if we still need the base attributes. Do you happen to have any knowledge on this?

Reply:
```suggestion
runtime-version: '24.08'
```